Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -136,11 +150,23 @@ class SentryTracerTests: XCTestCase {
             XCTAssertEqual(tracerTimestamp.timeIntervalSince1970, span["timestamp"] as? TimeInterval)
         }
     }
+
+    func testDeadlineTimer_FinishesTransactionAndChildren() {
```

Comment:
`h`: From reading the test code, I would assume that the timeout also works for non-auto-generated transactions. I would expect that the timeout only runs for auto-generated transactions. Please adapt this test case accordingly.

Reply:
The timeout does only work for auto-generated transactions. I don't understand your comment?

> Please adapt this test case accordingly

What needs to be adapted? The test case already tests the right thing.

Reply:
Ah sorry my bad, it wasn't clear to me that getSut returns an auto-generated transaction.